README.md

```diff
@@ -1,7 +1,25 @@
 # UnionPatcher
 
-A tool for LittleBigPlanet that patches the server urls to a custom url.
+A commandline tool that will take the official EBOOT of a LittleBIGPlanet title and replace the server URLs with the ones specified
```

Comment:
While UnionPatcher does have a command line utility, there *is* also a GUI.

Also, this sentence reads a bit worse in general.

Reply:
Removed the references to the commandline